### PR TITLE
Gci-Saad Khaleeq

### PR DESCRIPTION
**Pull request by saadkhaleeq610**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Gci-19 Saad",
+  "name": "reactapptask",
```
Comment (f4ww4z) [inline package.json]: Change this to a sentence describing what this specific app does.

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,16 @@
 import React from 'react';
-import logo from './logo.svg';
+import logo from './assets/img/code-inlogo.png';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
```
Comment (f4ww4z) [inline src/App.js]: Make this image horizontal.

File: package.json (hunk 1 above)
Comment (saadkhaleeq610) [reply package.json]: done

File: src/App.js (hunk 2 above)
Comment (saadkhaleeq610) [reply src/App.js]: done

**Comment by saadkhaleeq610** [comment]
reviewers requested to reopen the branch

**Pull request closed without merging**